manifests/repositories.pp

```diff
@@ -20,8 +20,8 @@
         }
       } else {
         yumrepo { 'htcondor-stable':
-          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${major_release}",
-          baseurl  => "http://research.cs.wisc.edu/htcondor/yum/stable/rhel${major_release}",
+          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${facts['os']['release']['major']}",
+          baseurl  => 'http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever',
```

Comment:
Hi, 
I can't reproduce your observation using an SL7 live CD. 
Due to the use of the single quotes, `$releasever` ends up in the `.repo`-file (puppet does not see it). Yum in turn interprets this as `7` on CentOS 7 and from my observation using an SL 7.3 Live CD, that is seems to be the case also for SL 7.3. 
Additionally, yum on SL also offers `$slreleasever` which indeed expands to "7.3". 
Can you confirm?